src/components/Calendar/hooks/useCalendarState.ts

```diff
@@ -194,6 +229,9 @@ export function useCalendarState(props: CalendarStateOptions): CalendarState {
         isWeekend(date: DateTime) {
             return this.mode === 'days' && [0, 6].includes(date.day());
         },
+        isCurrent(date: DateTime) {
+            return dateTime({timeZone}).isSame(date, this.mode);
```

Comment:
[isSame](https://github.com/gravity-ui/date-utils/blob/986b63f1996a86a57ce261739a27eb05a61ae2c4/src/dateTime/dateTime.ts#L234) does not depend on the time zone